Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN npm install
 
 COPY . .
 
-RUN npm run compile
+RUN npm run compile && npm test
```

Comment:
I think we should do this in the CI, and not in the Docker build process

Reply:
Doesn't this mean downloading all the `node_modules` twice ?
Also, we do this on [codacy-eslint](https://github.com/codacy/codacy-eslint/blob/1581024f3fd3187d4a27374c4ffe17c793b3dab7/Dockerfile#L11) and served us quite well.

Reply:
But it's running tests just to build a container image. Also, if you want to save some bucks, you can run these tests in a container. Instead of a machine. And developing locally is also easier.

Finally, the concerns are so different (ci -> validations and automation, docker -> get a runnable image) that people new to the repository might have difficulty navigating

> Also, we do this in codacy-eslint and served us quite well.

As a person that usually has to comment that line, I kinda disagree 😄

Reply:
Well, for me whatever, running in the CI wasn't slow IMHO.

Reply:
>Doesn't this mean downloading all the node_modules twice ?

Can we cache these?

Reply:
https://app.circleci.com/pipelines/github/codacy/codacy-markdownlint/20/workflows/88a34e89-ada4-4efb-b107-202f0e5b0593/jobs/57
Damn it was fast!
Ok, I'm going to port the changes to codacy-eslint :-D